From 4cacc136afea2b723fbc03730739923ef34a5908 Mon Sep 17 00:00:00 2001
From: Xianzhong <xianzhong.li@nxp.com>
Date: Mon, 3 Sep 2018 22:01:09 +0800
Subject: [PATCH 4593/5242] MGS-3940-2 gpu: imx: dpu-blit: fix command buffer
 mapping

commit  c31b7444224ad77d60717fe3244de5c696832818 from
https://source.codeaurora.org/external/imx/linux-imx.git

fix the hard-code(not aligned with command buffer size)

Signed-off-by: Xianzhong <xianzhong.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu-blit/dpu-blit.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/imx/dpu-blit/dpu-blit.c b/drivers/gpu/imx/dpu-blit/dpu-blit.c
index ef1e5b8..f33d163 100644
--- a/drivers/gpu/imx/dpu-blit/dpu-blit.c
+++ b/drivers/gpu/imx/dpu-blit/dpu-blit.c
@@ -388,7 +388,7 @@ int dpu_bliteng_init(struct dpu_bliteng *dpu_bliteng)
 	dpu_base = res->start;
 
 	/* remap with bigger size */
-	base = devm_ioremap(dpu->dev, dpu_base, 64*SZ_1K);
+	base = devm_ioremap(dpu->dev, dpu_base, COMMAND_BUFFER_SIZE);
 	dpu_bliteng->base = base;
 	dpu_bliteng->dpu = dpu;
 
-- 
1.7.9.5

